cmd/gossamer/gossamer_test.go

```diff
@@ -0,0 +1,154 @@
+package main_test
+
+import (
+	"encoding/json"
+	"fmt"
+	"go/build"
+	"os"
+	"os/exec"
+	"path"
+	"reflect"
+	"strings"
+	"testing"
+	"time"
+
+	cfg "github.com/ChainSafe/gossamer/config"
+	"github.com/ChainSafe/gossamer/p2p"
+	"github.com/ChainSafe/gossamer/polkadb"
+	"github.com/inconshreveable/log15"
```

Comment:
Lets alias this as `log`. This is something that should be added to our style guide, just as a means to stop people from accidentally using `log` package instead.